server/webapp/WEB-INF/rails/webpack/views/components/server_health_summary/server_health_messages_modal.tsx

```diff
@@ -58,7 +58,7 @@ export class ServerHealthMessagesModal extends Modal {
     return <li data-test-id={messageId}
                data-test-message-level={message.level.toLowerCase()}
                className={classnames(styles.serverHealthStatus, message.level.toLowerCase())}>
-      <span data-test-class="server-health-message_message" className={styles.message}>{message.message}</span>
+      <span data-test-class="server-health-message_message" className={styles.message}>{m.trust(message.message)}</span>
```

Comment:
Since most messages are already escaped in ServerHealthState and the messages where we want to show html are crafted by us, m.trust here should be fine.